src/controller/blob/controller.go

```diff
@@ -75,6 +75,9 @@ type Controller interface {
 
 	// GetAcceptedBlobSize returns the accepted size of stream upload blob.
 	GetAcceptedBlobSize(sessionID string) (int64, error)
+
+	// ReFreshUpdateTime updates the update time for the blob.
+	ReFreshUpdateTime(ctx context.Context, digest string, time time.Time) (err error)
```

Comment:
It should be `Refresh` not `ReFresh`, and if it's refresh is should always be `Now` correct?  Why it's a parm?

If we think about the flow `RefreshUpdateTime` is a side effect of the API call right?  Is that necessary to expose it on controller level?

Reply:
it's ok to remove the param from method and use now in the method.

To expose it in the ctl is because that it should be called in the middleware. And in the current program model, middleware/API does only consume ctl.

Reply:
the code is refactored, just refer to the latest commit.